src/main/webapp/app/app.scss

```diff
@@ -7,6 +7,10 @@ html {
   font-size: $oncokb-font-font-size;
 }
 
+a {
+  text-decoration: underline;
+}
```

Comment:
This will give underline to all a tag on the page. For instance, the navigation sidebar uses <a> tags as part of the NavLinks I believe. I think we only need it for reference linkouts?